.circleci/config.yml

```diff
@@ -96,6 +96,13 @@ references:
         gem install bundler -v $bundler_version
         bundle config --local path vendor/bundle
         bundle check || bundle install --jobs=4 --retry=3
+  _install-wiremock: &install-wiremock
+    run:
+      name: Install java and wiremock
+      command: |
+        sudo apt-get update
+        sudo apt-get install default-jre
+        curl -o spec/wiremock/wiremock-standalone.jar https://repo1.maven.org/maven2/com/github/tomakehurst/wiremock-standalone/2.27.1/wiremock-standalone-2.27.1.jar
```

Comment:
Is there a official docker image for this that we  could  run in  out test-executor bundle? 
It might speed up things as bit.

Reply:
So some of the DPS teams use this one - https://github.com/rodolpheche/wiremock-docker

Reply:
I've now updated this to use the dockerised wiremock in the test-executor